### ISSUE TITLE
custom vehicle movement inside parkingArea

### ISSUE THREAD

**Issue by summerbamboo** [eclipse/sumo#13509]
I am on the research of the vehicle scheduling method in the high density parking lot. I have created a high density parking lot in sumo as below. If the target vehicle(red square) wants to leave the parking lot but blocked by the vehicles above it(blue square), the blocking vehicles need to be moved out(to other parking spaces). Is there any chance that I can use any function to move the parked vehicles inside the parking lot?

<img width="651" alt="1687858111936" src="https://github.com/eclipse/sumo/assets/125885421/a29e879e-3850-4d2d-9894-41318163b956">

**Comment by namdre** [comment]
At the moment, no. The `<space>` objects only serve visualization and are not semantically accessible.  I think the best approach to model your project would be to define one parkingArea for each column of your high-density parking lot (each on a separate lane) and set them to `onRoad="True"`. This way you could move cars on actual lanes to reflect their position in the high-density lot (and "drive" them into other columns to free up the way. You can set their 'minGap' requirement to 0 to achieve the desired density.

**Comment by namdre** [comment]
please comment if something is still unclear

**Comment by summerbamboo** [comment]
Thanks for your advice! I've tried to create the high density parking lot on the lane, and it worked, but I don't know why the vehicles won't stop for the parking as I already set the `stop parkingArea` in the route file. i will try to figure this out later. Thans again!!!
